app/controllers/application_controller.rb

```diff
@@ -17,8 +17,8 @@ class ApplicationController < ActionController::Base
   def error_503(e); error(503, e); end
 
   def error(status_code, exception = nil)
-    if exception && defined? Airbrake
-      env["airbrake.error_id"] = notify_airbrake(exception)
+    if exception && defined? GovukError
+      GovukError.notify(exception.to_s)
```

Comment:
GovukError/Sentry can handle an exception object, which will make it report things like the error message, location and backtrace. Calling `.to_s` on it will just report the error message to Sentry.

Reply:
👍 perfect thanks